src/handleImportMap.ts

```diff
@@ -32,38 +43,67 @@ export async function handleImportMap(
 async function readImportMap({
   orgKey,
   importMapName,
-}: Params): Promise<ImportMap> {
-  let importMapKV: ImportMap | null = null;
+}: Params): Promise<ImportMap | null> {
+  const kvKey = `import-map-${orgKey}-${importMapName}`;
+
   try {
-    importMapKV = (await MAIN_KV.get(`import-map-${orgKey}-${importMapName}`, {
+    return (await MAIN_KV.get(kvKey, {
       type: "json",
     })) as ImportMap | null;
   } catch (err) {
     // Defensive programming so that we don't take down their production applications
     // if KV has invalid JSON in it or is down
+    console.error(`Error reading import map with key ${kvKey}`);
     console.error(err);
+    return null;
```

Comment:
How will we know if this is happening? Will console errors get logged somewhere? Will we be notified?

Reply:
Good question. The console.error on the line above will be something we can see in cloudflare's UI. But the full fledged solution would be one with proper monitoring and alarming where we get emails if there are lots of errors. I wasn't sure how to set that up so I thought writing to stderr is a decent start, since everything supports letting us see that.